docker-compose doesn't work unless I close chrome

## environment 

OS: MacOS 10.13.4
Docker for Mac: Version 18.03.0-ce-mac59 (23608)
Google Chrome is up to date: Version 65.0.3325.181 (Official Build) (64-bit)

## Steps to replicate:
```
git clone https://github.com/apache/incubator-openwhisk-devtools
cd incubator-openwhisk-devtools/docker-compose
make quick-start
Downloading source tar ball....
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   134    0   134    0     0    257      0 --:--:-- --:--:-- --:--:--   257
100 10.3M    0 10.3M    0     0  1442k      0 --:--:--  0:00:07 --:--:-- 1765k
Unpacking tarball.
downloading the CLI tool ...
downloading cli for mac
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   624    0   624    0     0   2455      0 --:--:-- --:--:-- --:--:--  2456
100 3352k  100 3352k    0     0  1184k      0  0:00:02  0:00:02 --:--:-- 1430k
Archive:  wsk.zip
  inflating: wsk
  inflating: NOTICE.txt
  inflating: README.md
  inflating: LICENSE.txt
host ip address: 172.17.0.1
checking required ports ...
Port 443 is taken by PID:8659.
make: *** [check-required-ports] Error 1
```
Then I shutdown the Chrome App try again `make quick-start` and it works

--- a/docker-compose/Makefile
+++ b/docker-compose/Makefile
@@ -131,7 +131,7 @@ print-host:
 .PHONY: check-required-ports
 check-required-ports:
 	echo "checking required ports ... "
-	for port in 80 443 2888 5984 8085 8888 9092 2888 8001; do \
+	for port in 2888 5984 8085 8888 9092 2888 8001; do \
 		pid=`lsof -Pi :$$port -sTCP:LISTEN -t` ; \
 		if [ ! -z "$$pid" ];  then echo "$$(tput setaf 1)Port $$port is taken by PID:$$pid.$$(tput sgr0)"; exit 1; fi; \
 	done

